src/components/Mapper/Trainers/TrainersModal.jsx

```diff
@@ -13,6 +14,33 @@ const TrainersModal = ({
   const handleClose = () => {
     onHide();
   };
+  const [showSnackbar, setShowSnackbar] = React.useState(false);
+  const handleExport = () => {
+    const showdownMon = selectedTrainer.team.map(pokemon => transformToText(pokemon));
+    const exportText = showdownMon.join("\n\n");
+    navigator.clipboard.writeText(exportText).then(() => {
+      setShowSnackbar(true);
+    });
+
+  };
+
+  const handleCloseSnackbar = () => {
+    setShowSnackbar(false);
+  };
+
+  function transformToText(pokemon) {
+    const genderSymbol = pokemon.gender === "FEMALE" ? "(F)" : "(M)";
```

Comment:
There are genderless pokemon referred to as "NEUTRAL" in the code. How would this work with them?

Reply:
i guess it would default to male, i checked showdown and it accepts not specifying for genderless pokemon. it's probably better to specify it for gendered mons because of rivalry, i'll add a check for that property